docs/en/ufs/MapR-FS.md

```diff
@@ -83,19 +83,25 @@ ${ALLUXIO_HOME}/bin/alluxio fs mount /<path in Alluxio>/ maprfs:///<path in MapR
 
 Start up Alluxio locally to see that everything works.
 
-{% include Common-Commands/start-alluxio.md %}
```

Comment:
The origin contents are located at https://github.com/Alluxio/alluxio/tree/master/docs/_includes/Common-Commands